homeassistant/components/glinet/config_flow.py

```diff
@@ -0,0 +1,186 @@
+"""Config flow for GL-inet integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from gli_py import GLinet
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.device_tracker.const import (
+    CONF_CONSIDER_HOME,
+    DEFAULT_CONSIDER_HOME,
+)
+from homeassistant.const import CONF_API_TOKEN, CONF_HOST, CONF_PASSWORD
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import format_mac
+
+from .const import DOMAIN
+
+# from homeassistant.helpers import config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+# TODO adjust the data schema to the data that you need
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default="http://192.168.8.1"): str,
+        vol.Required(CONF_PASSWORD, default="goodlife"): str,
```

Comment:
Are these defaults actual defaults, or only in your installation?

Reply:
These are the actual defaults for GLinet based routers - they were originally marketed a travel routers so used the uncommon .8 subnet as to avoid conflicts when used behind another local network

Reply:
The problem with using the default password is that a user by default cannot see what is entered in the field, so it's not that useful.